ParkingSlots.Tests/Controller/ParkingSlotControllerTests.cs

```diff
@@ -54,10 +54,11 @@ public void GivenParkingZoneId_WhenIndexIsCalled_ThenReturnsParkingSlotsVM()
         var expectedVMs = new List<ListOfParkingSlotsVM>();
         expectedVMs.AddRange(ListOfParkingSlotsVM.MapToModel(_parkingSlot));
 
-        _slotService.Setup(x => x.GetSlotsByZoneId(_Id)).Returns(_parkingSlot);
+        _slotService.Setup(x => x.GetSlotsByZoneId(_id)).Returns(_parkingSlot);
+        _zoneService.Setup(x => x.GetById(_id)).Returns(_parkingZone);
 
         //Act
-        var index = _controller.Index(_Id);
+        var index = _controller.Index(_id);
         var model = ((ViewResult)index).Model;
```

Comment:
In Act part why are you taking model
It should be in Assert part
var model = Assert.IsType(create).Model;